buckw

```diff
@@ -75,20 +75,26 @@ getChanges ( ) {
             ["suffix", "gradle"],
             ["name", ["gradle-wrapper.properties", "lint.xml"]],
             ["match", "**/src/**/AndroidManifest.xml", "wholename"]
+        ],
+        ["not",
+            ["dirname", ".okbuck"]
         ]
     ],
     "fields": ["name"]
 }]
 EOT`
     SOURCE_ROOTS=`watchman --output-encoding=json --no-pretty -j 2>&1 <<-EOT
-["query", ".", {
+["query", "$WORKING_DIR", {
```

Comment:
I'm not sure I totally follow. By excludes/includes do you mean in the context of watchman?

If need be, you can specify a `relative_root` for watchman in the case that you'd want `$WORKING_DIR` to differ from the watched root like I mentioned in the PR summary. Watchman includes/excludes are evaluated relative to the relative_root if specified (https://facebook.github.io/watchman/docs/file-query.html#relative-roots). I'm not making that change right now but it should be possible.